components/feature/contextmenu/src/main/res/layout/mozac_feature_contextmenu_dialog.xml

```diff
@@ -2,29 +2,37 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<ScrollView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:fadeScrollbars="false">
+    android:layout_height="match_parent">
 
-    <androidx.appcompat.widget.LinearLayoutCompat
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerView"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:scrollbars="vertical"
+        android:paddingBottom="16dp"
```

Comment:
Can we add `android:clipToPadding="false"` here to avoid the content being clipped when scrolling?
<img width="179" alt="image" src="https://user-images.githubusercontent.com/11428869/157618714-08f1aa09-e313-46fb-864a-296a17a5bc4a.png">

Reply:
Sure, will make the change